lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4300,6 +4366,7 @@ namespace {
     }
 
     Decl *VisitUsingShadowDecl(const clang::UsingShadowDecl *decl) {
+      // FIXME: understand this
```

Comment:
Probably not worth committing this FIXME :)

Reply:
Oops 👀 Thanks for spotting this!